prac_06/languages.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404 prac, week 6
+languages.py
+"""
+
+from prac_06.programming_language import ProgrammingLanguage
+
+
+def main():
+    python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
```

Comment:
In the lecture notes Lindsay has said that every function needs to have a docstring at the start of it.